docs/source/tutorials/faq.ipynb

```diff
@@ -0,0 +1,89 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ffe0d62e",
+   "metadata": {},
+   "source": [
+    "# FAQ\n",
+    "\n",
+    "Answers to frequently asked questions about the [cleanlab](https://github.com/cleanlab/cleanlab) open source package"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eca36874",
+   "metadata": {},
+   "source": [
+    "### How do I format labels for Cleanlab?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d5d0fbb3",
+   "metadata": {},
+   "source": [
+    "Cleanlab only works with integer-encoded labels in the range `{0,1, ... K-1}` where `K = number_of_classes`. Final labels array should only contain integer values in the range  `{0, K-1}` and be of shape `(N,)` where `N = total_number_of_data_points`.\n",
+    "\n",
+    "**Text labels** should to be integer-encoded. An example of text to integer-encoding can be found in step 2 of [Text Classification with TensorFlow, Keras, and Cleanlab Tutorial](https://docs.cleanlab.ai/v2.0.0/tutorials/text.html#2.-Load-and-preprocess-the-IMDb-text-dataset). \n",
+    "\n",
+    "**One-hot encoded labels** should be integer-encoded by finding the argmax along the one-hot encoded axis. An example of what this might look like is shown below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "239d5ee7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np \n",
+    "\n",
+    "arr  = np.array([[0,1,0,0],[1,0,0,0],[0,0,0,1]]) # [3,4] array wih 4 labels\n",
+    "one_hot_encoded_axis = 1\n",
+    "int_arr = np.argmax(arr, axis=one_hot_encoded_axis) # [3, ] array with label 0-3 assigned per row "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "49c3927b",
+   "metadata": {},
+   "source": [
+    "Here `arr` is the one-hot encoded array and `one_hot_encoded_axis` is the axis of the one-hot encoded feature columns. `int_arr` is the final integer-encoded array. After labels array is integer-encoded, see in [The Workflows of Data-Centric AI Tutorial](https://docs.cleanlab.ai/v2.0.0/tutorials/indepth_overview.html#Workflow-1:-Use-CleanLearning()-for-everything) how to pass it into the model. "
```

Comment:
I think we can omit this cell. It should be pretty obvious from the above how user can utilize `labels`. The FAQ answers are not intended to be full tutorials, just to help somebody get unstuck (and for cleanlab team members to be able to quickly point to a link when users ask questions!)